server/public/partials/statblock/statBlock.html

```diff
@@ -152,13 +152,12 @@ <h4>Languages</h4>
           <span ng-repeat="language in creature.stats.languages"><span ng-if="$index>0">, </span>{{language}}</span>
         </p>
       </div>
-      <div class="property-line">
-        <h4 style="opacity: 0;">Challenge</h4>
-        <div class="property-line" style="float: left; margin-left: -3.5%">
+      <div class="property-line" style="margin-bottom: 1.33em">
```

Comment:
added a margin here to ensure text has some distance from the separator line below